pkg/sql/template_elasticdl.go

```diff
@@ -107,16 +107,16 @@ class PredictionOutputsProcessor(BasePredictionOutputsProcessor):
         if all(
             k in os.environ
             for k in (
-                "MAXCOMPUTE_PROJECT",
-                "MAXCOMPUTE_AK",
-                "MAXCOMPUTE_SK",
+                "SQLFLOW_TEST_DB_MAXCOMPUTE_PROJECT",
```

Comment:
It seems these are not for test, is it?

Reply:
Great catch!

Reply:
@terrytangyuan Should I revert these changes in `template_elasticdl.go`?

Reply:
Yes, revert them for now. We need to change the namings in ElasticDL accordingly in order to make this codegen work.